tasks/updateconstants.js

```diff
@@ -669,6 +676,106 @@ const sources = [
       return result;
     },
   },
+  {
+    key: "aghs_desc",
+    url: aghs_desc_urls,
+    transform: (respObj) => {
+      const herodata = respObj;
+      aghs_desc_arr = [];
+
+      // for every hero
+      for(i=0;i < herodata.length; i++)
+      {
+        // check validity of data from URL
+        // some hero IDs are unused (to the public ;) )
+        if(herodata[i].result.data.heroes.length == 0 || herodata[i].result.data.heroes[0].abilities.length == 0)
```

Comment:
You can remove this if you just use the list of heroes

Reply:
agreed